src/components/10-atoms/input-text/index.scss

```diff
@@ -80,20 +80,36 @@ axa-input-text {
 
   .a-input-text__error {
     @include typo();
-    margin-top: 10px;
+    display: block;
```

Comment:
use `$border-width` and `$border-type` on line 23

Reply:
fix in c9feb9c